src/frontend/components/InstructorView/index.tsx

```diff
@@ -16,8 +16,7 @@ const messages = defineMessages({
     id: 'components.InstructorView.btnDashboard',
   },
   disabledDashboard: {
-    defaultMessage:
-      'This video is imported from another playlist. You can go to the original playlist to directly modify this video, or delete it from the current playlist and replace it by a new video.',
+    defaultMessage: 'This video belongs to {context_id}',
```

Comment:
I agree that the old message was probably too specific for regular users, but is this very helpful? What does a `{context_id}` look like in real use?

Reply:
it looks like something like this: `course-v1:test+test+test`

Reply:
It refers to the course and the section the original video belongs to.

Reply:
Ok so the user can understand the reference, right? What about a message like this:

```suggestion
    defaultMessage: 'This video is read-only because it belongs to another course: {context_id}',
```

Reply:
I think we should not mention `read-only` they have no idea about this. So I think this message should be better: `This video belongs to the course: {context_id}`

Reply:
But isn't the whole reason we need to show this message that they cannot modify it? Maybe I'm just misunderstanding why we need this message

Reply:
Yes in fact you are maybe right, I will add `read-only`

Reply:
Done